content/docs/developing-operators/tasks.md

```diff
@@ -177,13 +177,97 @@ Once our Nginx server is up and running we should be able to read a pearl of cow
 
 **Note:**
 
-- File generating Pod has to be side-effect free (meaning side-effects that are observable outside of the container like a 3rd party API call) as the container might be executed multiple times on failure. A `restartPolicy: OnFailure` is used for the pipe-pod
+- File generating Pod has to be side effect free (meaning side effects that are observable outside of the container like a 3rd party API call) as the container might be executed multiple times on failure. A `restartPolicy: OnFailure` is used for the pipe-pod
 - Only files <1Mb are applicable to be stored as ConfigMap or Secret. A pipe-task will fail should it try to copy files >1Mb
 - As of KUDO 0.9.0 pipe-artifacts can only be used within the same plan they were generated
 - You can pipeline artifacts by creating them in one pipe-task and mounting in a subsequent one
 
 Full cowsay-operator can be found in the [KUDO operators repo](https://github.com/kudobuilder/operators/tree/master/repository/cowsay).
 
+## KudoOperator-Task
+
+As of KUDO 0.15.x, KUDO supports a `KudoOperator` task which allows you to specify a dependency on other operators. While dependencies in general is a complicated topic, `KudoOperator` itself is about installation dependencies i.e. your operator instance and all its dependencies (including transitive ones) will be installed and/or removed as one unit.
+
+KUDO operators already have a mechanism to deal with installation dependencies called [plans, phases, and steps](plans.md) with serial or parallel execution strategy. This mechanism is already powerful enough to express any dependency hierarchy including transitive dependencies.
+
+Here a simple example of a task specifying a dependency on the community Zookeeper operator version 0.3.0 (which will install [zookeeper 3.4.14](https://github.com/kudobuilder/operators/blob/master/repository/zookeeper/README.md))
+
+```yaml
+tasks:
+  - name: deploy-zookeeper
+    kind: KudoOperator
+    spec:
+      package: zookeeper
+      operatorVersion: 0.3.0
+```
+
+As with other tasks, KUDO will make sure this task is healthy before moving to the next one. In the example below we split the `deploy` plan into two phases: _prereqs_ and _main_:
+ 
+```yaml
+plans:
+  deploy:
+    strategy: serial
+    phases:
+      - name: prereqs
+        strategy: parallel
+        steps:
+          - name: first
+            tasks:
+              - deploy-zookeeper
+      - name: main
+        strategy: parallel
+        steps:
+          - name: second
+            tasks:
+              - deploy-main
+```
+
+"Healthy", in the case of the operator means that its `deploy` plan has finished successfully. `KudoOperator` closely mimics the `kubectl kudo install` command semantics allowing to additionally specify the `appVersion` (defaults to most recent one) and the `instanceName` (will be generated by KUDO by default).
```

Comment:
it might be valuable here to use the word "complete" here... the current sentence requires some translation or deeper understand that shouldn't be necessary.  The "deploy" plan has finished succcessfully...  how does the reader know what that means?   According to the `plan status` it means the plan is "Complete"